### ISSUE TITLE
Context menu to open external application

### ISSUE THREAD

**Issue by anffna** [fooyin/fooyin#226]
Hey man, great software! So, could you provide a script that calls external software to edit tags?

**Comment by alekksander** [comment]
yes, there could be rmb position to "open with…" which could work with various preferable apps (tagging, spectrum analyzers, console).
